src/main/cpp/src/utils/vid_mapper.cc

```diff
@@ -100,7 +100,8 @@ std::unordered_map<std::string, int>({
   {"element_wise_sum", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_ELEMENT_WISE_SUM},
   {"elementwise_sum", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_ELEMENT_WISE_SUM},
   {"concatenate", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_CONCATENATE},
-  {"histogram_sum", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_HISTOGRAM_SUM}
+  {"histogram_sum", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_HISTOGRAM_SUM},
+  {"ignore", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_IGNORE}
```

Comment:
As part of #71, we have introduced "none" - see Line 95 - {"none", VCFFieldCombineOperationEnum::VCF_FIELD_COMBINE_OPERATION_NONE}. Is the "ignore" functionality different from "none"

Reply:
Yeah, this was a commit parallel to your PR - the subsequent commit in this PR rolls back that change.